samcli/local/apigw/local_apigw_service.py

```diff
@@ -142,10 +144,15 @@ def _request_handler(self, **kwargs):
         Response object
         """
         route = self._get_current_route(request)
+        cors_headers = LocalApigwService.cors_to_headers(route.cors)
```

Comment:
If `cors_to_headers` returned an empty dict instead of None, then all the if checks on cors_headers go away and makes the code cleaner. Avoiding None unless a specific reason is not a bad practice to have. :)